pyRMSD/symmTools.py

```diff
@@ -17,6 +18,22 @@ def symm_groups_validation( symm_groups):
                     raise Exception
     except Exception:
         raise ValueError('Symmetry groups are not well defined')
+        
+        
+def symm_groups_validation_new( symm_groups):
```

Comment:
The name of the function - or at least the docstring - should clarify how it differs from the original `symm_groups_validation`.

Reply:
Added in the docstring